### PR TITLE
mention Ali's "Programming in D" prominently on "Gettting Started"

### PR DESCRIPTION
**Pull request by aG0aep6G**
We should take newbies by the hand on "Getting Started". And as far as I'm
aware, Ali's book is the best thing we have in that category.

Also, drive-by fix of bad parenthesis.

### CODE REVIEW

File: getstarted.dd [hunk 1]
```diff
@@ -4,8 +4,14 @@ $(D_S
 Getting Started,
 
   $(P
-  $(B D) is a powerful systems programming language. The
-  best places to learn are:)
+  $(LINK2 http://ddili.org/ders/d.en/index.html, $(B Programming in D)) is an
+  extensive tutorial by Ali Çehreli and a great starting point for absolute
+  newbies. It doesn't assume that you know any other programming languages, and
```
Comment (JinShil) [inline getstarted.dd]: Prefer the term "beginners" over "newbies".

Comment (aG0aep6G) [inline getstarted.dd]: I like "newbies". But I wouldn't mind changing it to "beginners".

Comment (andralex) [inline getstarted.dd]: yah, let's be conservative at first - beginners or newcomers should be fine

**Comment by andralex** [comment]
change the word and let's pull please - thanks!

**Comment by aG0aep6G** [comment]
updated to "beginners"

**Comment by aG0aep6G** [comment]
Ping?

**Pull request merged**